### DISCUSSION TITLE
ntpchecker

### DISCUSSION THREAD

**Discussion by ThomasCFelix** [R00tS3c/DDOS-RootSec#16] [Q&A]
where is ./ntpchecker?

**Comment by R00tS3c** [comment]
Hi, sorry for waiting, pastebin has removed all old scripts i have them all compiled but i will try to find again all sources and push them, if you dont want to wait join on telegram i will send you.

**Comment by R00tS3c** [comment]
You can now find all here https://github.com/R00tS3c/DDOS-RootSec/tree/master/DDOS%20Scripts/AMP%20Methods

**Comment by ThomasCFelix** [comment]
Thanks but I have not found the ntpchecker yet.
screen ./ntpchecker monlist_fingerprint.txt step1.txt 1 0 1

**Comment by R00tS3c** [comment]
https://github.com/R00tS3c/DDOS-RootSec/blob/master/DDOS%20Scripts/AMP%20Methods/NTP%20-%20SNMP%20-%20HAVEN%20-%20DNS%20-DRDOS%20-%20FRAG%20-%20SUDP%20-%20MEMCACHED/ntp_scanner.c

**Comment by ThomasCFelix** [comment]
ntp_scanner.c is different
 ./ntpchecker monlist_fingerprint.txt step1.txt 1 0 1 | has two texts in the parameters but ntp_scanner.c is something else.
Also Fud-Bypass.c seems to have a problem, after capturing the packet with tcpdump i see this incorrect payload 
%randompayload%randompayload%randompayload%%randompayload%%randompayload%%randompayload%%randompayload%%randomp.....................................................................
**Cannot generate random payload.**
